Azkfile.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Documentation: http://docs.azk.io/Azkfile.js
+ */
+
+// Adds the systems that shape your system
+systems({
+  turniquette: {
+    // Dependent systems
+    depends: [],
+    // More images:  http://images.azk.io
+    image: {"dockerfile": "."},
+    // Steps to execute before running instances
+    provision: [
+      "bundle install --path .bundle",
+      "bundle exec rake db:create",
```

Comment:
I think this should not be done in provision as it'll be done frequently.
I prefer doing it by hand from the shell the first time.